forge-gui/res/cardsfolder/upcoming/ballroom_brawlers.txt

```diff
@@ -0,0 +1,14 @@
+Name:Ballroom Brawlers
+ManaCost:3 W W
+Types:Creature Human Warrior
+PT:3/5
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigSelectTargetCreature | TriggerDescription$ Whenever CARDNAME attacks, CARDNAME and up to one other target creature you control both gain your choice of first strike or lifelink until end of turn.
+SVar:TrigSelectTargetCreature:DB$ Pump | ValidTgts$ Creature.YouCtrl+Other | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one other target creature | AITgts$ Creature.withoutFirst Strike | SubAbility$ DBKeywordChoice | SpellDescription$ CARDNAME and up to one other target creature you control both gain your choice of first strike or lifelink until end of turn.

```

Comment:
This is here because I couldn't get it to work with the targeting on the GenericChoice - it worked perfectly for me, but the AI ignored the trigger when it had 2 or more creatures.

Reply:
Targeting before the choice seems right as I read the card... this might be one to revisit once they share the rules updates

Without digging into it too much, I think GenericChoice is more geared toward target/defined player making a choice that target/defined cards, so that's probably where the hole is in the AI code

@Agetian or @tool4ever - thoughts on this AILogic?

Reply:
@tool4ever I agree 100% - I made an issue